core/unix/os.c

```diff
@@ -8869,7 +8882,8 @@ get_application_base(void)
         const char *name = get_application_name();
         if (name != NULL && name[0] != '\0') {
             DEBUG_DECLARE(int count =)
-            memquery_library_bounds(name, &executable_start, &executable_end, NULL, 0);
+            memquery_library_bounds(name, &executable_start, &executable_end, NULL, 0,
+                                    NULL, 0);
             ASSERT(count > 0 && executable_start != NULL);
         }
 #    else
```

Comment:
I thought this would replace the other uses of `DYNAMORIO_LIBRARY_NAME` and remove the define?

(I assumed the issue covered that.  If that is too involved this would become incremental for that issue and no longer `Fixes`.)

Reply:
Ok let me look at that.

Reply:
I have removed DYNAMORIO_LIBRARY_NAME, but only on Unix.